Dockerfile

```diff
@@ -1,72 +1,48 @@
 FROM ubuntu
 USER root
 
-WORKDIR /home
-
-VOLUME ["/asn1_codec"]
-
-RUN export LD_LIBRARY_PATH=/usr/local/lib
-RUN export CC=gcc
+WORKDIR /asn1-codec
```

Comment:
Please rename it to asn1_codec to match the name of the repository or call it acm. asn1_codec is too close to asn1-codec and could cause spelling errors.